File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    var num1 =0;
    var num2 =0;
    for(var i in num_list){
        if(num_list[i] % 2 == 1){
            num1 = num1*10 + num_list[i];
        }else{
            num2 = num2*10 + num_list[i];
        }
    }
    return num1+num2;
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    num1 = ''
    num2 = ''
    for i in num_list:
        if i % 2 == 0:
            num1 += str(i)
        else:
            num2 += str(i)
    return int(num1) + int(num2)